runtime/src/bank.rs

```diff
@@ -6661,6 +6661,17 @@ impl Bank {
                 &self.runtime_config.compute_budget.unwrap_or_default(),
                 false, /* debugging_features */
             ));
+
+        // genesis_config loaded by accounts_db::open_genesis_config() from ledger
+        // has it's lamports_per_signature set to zero; bank sets its value correctly
+        // after the first block with a transaction in it. This is a hack to mimic
+        // the process.
+        let derived_fee_rate_governor =
+            FeeRateGovernor::new_derived(&genesis_config.fee_rate_governor, 0);
+        // new bank's fee_structure.lamports_per_signature should be inline with
+        // what's configured in GenesisConfig
+        self.fee_structure.lamports_per_signature =
+            derived_fee_rate_governor.lamports_per_signature;
```

Comment:
currently behavior: 
1. genesis_config either have fee_rate_gonervor set to 0 (or any other value) lamports_per_signature for test or 5_000 in production;
2. fee_structure is always default to 5_000 lamports_per_signature regardless.

This change is to initialize fee_structure to have same lamports_per_signature as in fee_rate_governor.